src/HttpClient.php

```diff
@@ -62,8 +62,6 @@ public function execute($method, $urlTemplate, $params = [], $headers = [], $bod
 	// https://github.com/guzzle/guzzle/blob/master/src/Utils.php#L93
 	$url = \GuzzleHttp\UriTemplate\UriTemplate::expand($urlTemplate, $params);
 
-  $url = "https://store.meplato.com/api/v2/catalogs?skip=0&take=0&sort=-created%2Cname";
```

Comment:
Wow, really. I did that...? That's not my code 🏃 lol

Reply:
No thats on me. I commited it by mistake after changing to guzzlehttp/uri-template